src/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+}
+
+.box-container {
+  display: flex;
+  flex-wrap: wrap ;
+}
+
+.box {
+  height: 300px;
+  min-width: 300px;
+  font-family: "Arial", serif;
+  font-size: 100px;
+  color: white;
+  flex: 33% 1 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.box-1 {
+  background-color: rgb(255,0,0);
+}
+
+.box-2 {
+  background-color: rgb(204,0,0);
+}
+
+.box-3 {
+  background-color: rgb(153,0,0);
+}
+
+.box-4 {
+  background-color: rgb(102,0,0);
+}
+
+.box-5 {
+  background-color: rgb(51,0,0);
+}
+
+.box-6 {
+  background-color: rgb(0,0,0);
+}
+
+@media screen and (min-width: 599px) {
+  .box-1 {
+    background-color: rgb(255,0,0);
+  }
+
+  .box-2 {
+    background-color: rgb(204,0,0);
+
+  }
+
+  .box-3 {
+    background-color: rgb(153,0,0);
+    order: 2;
+  }
+
+  .box-4 {
+    background-color: rgb(102,0,0);
+    order: 1;
+  }
+
+  .box-5 {
+    background-color: rgb(51,0,0);
+    order: 3;
+  }
+
+  .box-6 {
+    background-color: rgb(0,0,0);
+    order: 4;
+  }
```

Comment:
in different media queries only order properties are changing. Remove bg-color from media quieries